cumulusci/cli/service.py

```diff
@@ -283,7 +283,10 @@ def service_info(runtime, service_type, service_name, plain):
         service_data = [["Key", "Value"]]
         service_data.extend(
             [
-                [click.style(k, bold=True), str(v)]
+                [
+                    click.style(k, bold=True),
+                    (v[:5] + (len(v[5:]) * "*") if k == "token" else str(v)),
```

Comment:
Hard-coding the name here will cure the immediate P1 bug, but it's fragile and breaks the abstraction for Services. 

I suggest instead that we get the project config (`runtime.project_config or runtime.universal_config`). Then we can get the service definition (see `cumulusci/cumulusci.yml` around line 1306 and `service.py` around line 27-68) and find the `attribute` for this key. 

We can add to each attribute a new k/v pair in YAML: `secure: bool`.

Then we mask any attribute marked secure, which should include the GitHub token but also any future service with the same issue.

Reply:
OK, you pushed a commit doing essentially this within seconds of me submitting the review. Nice, @bethbrains